h2o-py/h2o/automl/autoh2o.py

```diff
@@ -273,6 +276,11 @@ def assert_is_step(s):
                             plan.append(dict(name=name, steps=[dict(id=i) for i in ids]))
             self.build_models['modeling_plan'] = plan
 
+        if monotone_constraints is not None:
+            if algo_parameters is None:
+                algo_parameters = {}
+            algo_parameters['monotone_constraints'] = monotone_constraints
+
```

Comment:
Py+R clients don't need the REST API changes, they can just use the `algo_parameters` logic.